site/docs/reference/Connectors/materialization-connectors/Snowflake.md

```diff
@@ -265,9 +269,14 @@ This is because most materializations tend to be roughly chronological over time
 This means that updates of keys `/date, /user_id` will need to physically read far fewer rows as compared to a key like `/user_id`,
 because those rows will tend to live in the same micro-partitions, and Snowflake is able to cheaply prune micro-partitions that aren't relevant to the transaction.
 
+### Snowpipe Streaming
+
+[Snowpipe Streaming](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-streaming-overview) is the lowest-latency method to load data into Snowflake.
+Snowpipe Streaming is used by default for [delta updates](#delta-updates) bindings. This method of ingress writes rows directly to Snowflake tables and scales compute automatically based on load.
+
 ### Snowpipe
 
-[Snowpipe](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-intro) allows for loading data into target tables without waking up the warehouse, which can be cheaper and more performant. Snowpipe can be used for delta updates bindings, and it requires configuring your authentication using a private key. Instructions for configuring key-pair authentication can be found in this page: [Key-pair Authentication & Snowpipe](#key-pair-authentication--snowpipe)
+[Snowpipe](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-intro) allows for loading data into target tables without waking up the warehouse, which can be cheaper and more performant. Snowpipe can be used for delta updates bindings; it requires configuring your authentication using a [private key](#key-pair-authentication) and using the `no_snowpipe_streaming` feature flag.
```

Comment:
I would probably just replace the `Snowpipe` section with the `Snowpipe Streaming` section entirely. I wouldn't expect users to ever set the `no_snowpipe_streaming ` flag.